### DISCUSSION TITLE
PreRecorded API: make a method that returns full list of supported audio formats

### DISCUSSION THREAD

**Discussion by grantdiamond** [deepgram/community#526] [Feature Requests]
I would love an API call that returns a full list of supported audio formats. We could ping this API once a day, for example, and make changes to the files we support in our app as we become notified of changes at deepgram. This would enable us to generate more business for deepgram, since right now we are rejecting all file types that aren't mentioned here: https://developers.deepgram.com/docs/supported-audio-formats

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jkroll-deepgram** [comment]
Hi @grantdiamond, note that the doc you linked mentions:
> Deepgram supports over 100 different audio formats and encodings. 

If you've come across any file formats that are causing errors, please feel free to share the file format(s) and request ID(s). Are there certain formats that you're receiving often that aren't on that list?

We work hard to ensure that the vast majority of audio formats are supported! Point taken about the convenience of that API - though we do not have an exhaustive list, and given the wide variety of different audio formats out there, it would be a slippery task to attempt.

Another option is rather than rejecting all file types besides the ones mentioned on our doc, you could convert yourself, such as with the CLI tool `ffmpeg`. For instance, to convert from mp4 (which we do also support) to wav (one of the most common lossless audio formats):
```
ffmpeg -i input.m4a -vn output.wav
```
